lib/components/src/controls/Boolean.tsx

```diff
@@ -80,16 +81,23 @@ const format = (value: BooleanValue): string | null => (value ? String(value) :
 const parse = (value: string | null) => value === 'true';
 
 export type BooleanProps = ControlProps<BooleanValue> & BooleanConfig;
-export const BooleanControl: FC<BooleanProps> = ({ name, value, onChange, onBlur, onFocus }) => (
-  <Label htmlFor={name} title={value ? 'Change to false' : 'Change to true'}>
-    <input
-      id={name}
-      type="checkbox"
-      onChange={(e) => onChange(e.target.checked)}
-      checked={value || false}
-      {...{ name, onBlur, onFocus }}
-    />
-    <span>False</span>
-    <span>True</span>
-  </Label>
-);
+export const BooleanControl: FC<BooleanProps> = ({ name, value, onChange, onBlur, onFocus }) => {
+  const onSetFalse = useCallback(() => onChange(false), [onChange]);
+  if (value === undefined) {
```

Comment:
What about `null`?

Reply:
Yes, sorry this PR was supposed to be a draft, and was based on other PRs that handled `undefined.

It doesn't actually handle incorrect arg values yet -- e.g. `null` or the wrong type.